xml/System/Decimal.xml

```diff
@@ -4695,7 +4695,7 @@
 |<xref:System.Globalization.NumberStyles.AllowThousands>|The *,* element.|  
 |<xref:System.Globalization.NumberStyles.AllowCurrencySymbol>|The *$* element.|  
 |<xref:System.Globalization.NumberStyles.Currency>|All. The `s` parameter cannot represent a hexadecimal number or a number in exponential notation.|  
-|<xref:System.Globalization.NumberStyles.Float>|The *ws* element at the beginning or end of `s`, *sign* at the beginning of `s`, and the *.*symbol. The `s` parameter can also use exponential notation.|  
+|<xref:System.Globalization.NumberStyles.Float>|The *ws* element at the beginning or end of `s`, *sign* at the beginning of `s`, and the *.* symbol. The `s` parameter can also use exponential notation.|  
```

Comment:
```suggestion
|<xref:System.Globalization.NumberStyles.Float>|The *ws* element at the beginning or end of `s`, *sign* at the beginning of `s`, and the \*.\* symbol. The `s` parameter can also use exponential notation.|  
```
There's also one other occurrence of the dropped \*.\* in the file.

Reply:
That would make the * appear. This is bolding the period.